### PR TITLE
making more attractive the 404 page

### PR DESCRIPTION
**Pull request by hksmith**
### Abstract
just modify the 404 page to make it more attractive and less annoying to see

### Motivation
I know that 404 pages make you feel bad when they appear on the screen so a least i make them less scary

### Specification
just add some CSS and image

### Backwards Compatibility
No backward compatibility issues found.

### Test Cases
I run it. and i worked!

### Reference Implementation
no just simple CSS code

### Security Considerations
NO need to discus just edit a page

### Copyright
Copyright and related rights waived via [CC0](https://github.com/ethereum/EIPs/blob/LICENSE.md).

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -4,20 +4,45 @@
...
+    font-size: 1.2em;
+    font-family: 'Arial', sans-serif;
+    color: #555555;
+    text-decoration: underline;
+    margin-top: 50px;
   }
 </style>
 
 <div class="container">
+  <img class="error-image" src="http://tsagi.com/upload/medialibrary/155/1553924c996c252bcbe450d8215967d6/ebe2ba14a34f64b897fd175631d8ce3c.jpg" alt="404 error illustration">
```
Comment (Pandapip1) [inline 404.html:43]: No external links.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: 404.html (hunk 1 above)
Comment (hksmith) [reply 404.html:43]: oh... so should i add the image directly

Comment (Pandapip1) [reply 404.html:43]: Yes. Please create a new images folder in the assets folder.

**Comment by hksmith** [comment]
I made the change by changing the image link to local link

**Comment by hksmith** [comment]
Am waiting to revised by you... am done with the requested change... I would apricate that you if you review and give me a response

**Review by Pandapip1 — APPROVED** [review]
LGTM

**Comment by hksmith** [comment]
The bot is not reviewing the request what should i do...

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
Oh, it's just waiting for more people to review.

**Review by lightclient — CHANGES_REQUESTED** [review]
I don't like the 404 image

**Comment by Pandapip1** [comment]
> I don't like the 404 image

It's still better than no image IMO.

**Comment by lightclient** [comment]
We shouldn't have a kitschy skeuomorphic 404 image. If you want a 404 image, take inspiration from the eth org website or general themes in the ethereum community.

**Pull request closed without merging**